frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePickerHeader.styled.tsx

```diff
@@ -22,12 +22,16 @@ export const TabButton = styled(Button)<TabButtonProps>`
   padding-right: 0;
   margin-left: ${space(2)};
   margin-right: ${space(2)};
-  border-bottom: ${({ selected }) =>
+  ${({ selected }) =>
     selected
-      ? "2px solid var(--mb-color-text-brand)"
-      : "2px solid transparent"};
-  color: ${({ selected }) =>
-    selected ? "var(--mb-color-text-brand)" : color("text-medium")};
+      ? css`
+          border-bottom: 2px solid var(--mb-color-text-brand);
+          color: var(--mb-color-text-brand);
+        `
+      : css`
+          border-bottom: 2px solid transparent;
+          color: var(--mb-color-text-secondary);
```

Comment:
Also, fix the button text color. It was `text-medium` before.

![Screenshot 2024-08-01 at 2 52 23 PM](https://github.com/user-attachments/assets/bc08aadf-e324-4151-8fad-cd3464335b31)